src/main/java/jme3test/jaimesascent/ui/Window.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2024 jMonkeyEngine
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ * * Redistributions of source code must retain the above copyright
+ *   notice, this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright
+ *   notice, this list of conditions and the following disclaimer in the
+ *   documentation and/or other materials provided with the distribution.
+ *
+ * * Neither the name of 'jMonkeyEngine' nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package jme3test.jaimesascent.ui;
+
+import com.jme3.app.Application;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
+import com.simsilica.lemur.Container;
+import com.simsilica.lemur.Panel;
+import com.simsilica.lemur.anim.Animation;
+import com.simsilica.lemur.anim.PanelTweens;
+import com.simsilica.lemur.anim.TweenAnimation;
+import com.simsilica.lemur.anim.Tweens;
+import com.simsilica.lemur.core.AbstractGuiControlListener;
+import com.simsilica.lemur.core.GuiControl;
+import com.simsilica.lemur.effect.AbstractEffect;
+import com.simsilica.lemur.effect.Effect;
+import com.simsilica.lemur.effect.EffectInfo;
+import e.g.jme3hudl.ControlLayout;
```

Comment:
I understand this is your own library, but for the sake of keeping it simple and easy for new jme users and not having them have to learn or use too many libraries, I'm going to ask this:
Would it be possible to do this with native lemur and not include another library?

My goal is to keep the project as flat and easily grasped as possible for the sdk. It's not really meant to be a showcase for advanced features.

Reply:
> I understand this is your own library, but for the sake of keeping it simple and easy for new jme users and not having them have to learn or use too many libraries, I'm going to ask this:

Actually, this library is nothing special, it just implements some "Designs" directly from Lemur.

> Would it be possible to do this with native lemur and not include another library?

If you don't want to include "third party" libraries, I can extract the class that implements this design (it can serve as an example for users on how to implement their own designs).

> My goal is to keep the project as flat and easily grasped as possible for the sdk

I understand that to use it as an SDK example the project has to be simple (work on possible changes).

> It's not really meant to be a showcase for advanced features.

Well, implementing a dynamic UI (as good as possible) with Lemur always requires using some advanced stuff (nothing a Java programmer can't do or understand), as leaving it so simple and straightforward limits a good impression, but if you want to make the example code as simple and understandable as possible for a newbie, I'll try to keep it as simple as possible...